### DISCUSSION TITLE
After 9 months, We CANNOT pull public images from the GitHub Package Registry without AUTH

### DISCUSSION THREAD

**Discussion by draveness** [community/community#26993] [Actions]
<p>The previous discussion was marked as solved, but I don’t think authentifcation is required when pulling from a public registry.</p>
<p><a href="https://github.community/t/docker-pull-from-public-github-package-registry-fail-with-no-basic-auth-credentials-error/16358">https://github.community/t5/GitHub-Actions/docker-pull-from-public-GitHub-Package-Registry-fail-with-quot/m-p/32782#M1247</a></p>

**Comment by chenrui333** [comment]
<p>This kind of reminds me the infamous docker ce download issue, <a href="https://github.com/docker/docker.github.io/issues/6910" rel="nofollow noopener">https://github.com/docker/docker.github.io/issues/6910</a> (Download Docker CE without logging in)</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/draveness">@draveness</a> ,</p>
<p>In a process, you currently need to login the GitHub Packages at least once before publishing/downloading/deleting a package from the GitHub Packages.</p>
<p>Even if packages are public, you still need to login the GitHub Packages at first.</p>
<p>If your projects really need the feature that public packages no need authenticating, I recommend you directly report a feature request <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=packages" rel="nofollow noopener">here</a>. That will allow you to directly interact with the appropriate engineering team, and make it more convenient for the engineering team to collect and categorize your suggestions.</p>
